newrelic.js

```diff
@@ -44,6 +44,38 @@ exports.config = {
    * attributes include/exclude lists.
    */
   allow_all_headers: true,
+  application_logging: {
+    /**
+     * If this is disabled, all sub-features are disabled.
+     * If it is enabled, the individual sub-feature configurations take effect.
+     */
+    enabled: true,
```

Comment:
this is currently defaulting to true in code there's no reason to set this in here